TestRunner/BuildTests.swift

```diff
@@ -53,29 +52,40 @@ class BuildTests {
         task.waitUntilExit()
     }
     
-    func addPlistEntry(name: String, value: String) {
-        let task = Process()
-        task.launchPath = "/bin/sh"
-        let infoPlist = String(format: "%@/%@.app/Info.plist", AppArgs.shared.derivedDataPath, AppArgs.shared.scheme)
-        let arguments = ["/usr/libexec/PlistBuddy", infoPlist, "-c", "\"Add :\(name) string \(value)\""]
-        task.arguments = ["-c", arguments.joined(separator: " ")]
-        task.standardError = Pipe()
-        task.standardOutput = Pipe()
-        task.launch()
-        task.waitUntilExit()
+    func addPlistEntries(name: String, value: String) {
```

Comment:
Instead of adding the regex and looping inside each of these functions, just change the `name: String` to `plistPath: String` and get the list of plist files up in `func build(listTests: Bool) throw` and iterate of the list and call the add/delete to plist functions from that loop